Crash when using eslint

### Describe the bug

The eslint watcher causes the Vite server to crash when it contains eslint warnings

The following error message appears:
```
$ vite

  VITE v4.1.4  ready in 1110 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  press h to show help

[vue-tsc] Found 0 errors. Watching for file changes.

node:internal/event_target:1006
  process.nextTick(() => { throw err; });
                           ^
TypeError [Error]: Cannot read properties of undefined (reading 'length')
    at getMarkerLines (/[PATH_TO_PROJECT]/node_modules/@babel/code-frame/lib/index.js:58:53)
    at codeFrameColumns (/[PATH_TO_PROJECT]/node_modules/@babel/code-frame/lib/index.js:100:7)
    at createFrame (/[PATH_TO_PROJECT]/node_modules/vite-plugin-checker/dist/cjs/logger.js:137:56)
    at /[PATH_TO_PROJECT]/node_modules/vite-plugin-checker/dist/cjs/logger.js:285:23
    at Array.map (<anonymous>)
    at normalizeEslintDiagnostic (/[PATH_TO_PROJECT]/node_modules/vite-plugin-checker/dist/cjs/logger.js:262:30)
    at /[PATH_TO_PROJECT]/node_modules/vite-plugin-checker/dist/cjs/checkers/eslint/main.js:122:91
    at Array.map (<anonymous>)
    at Object.configureServer (/[PATH_TO_PROJECT]/node_modules/vite-plugin-checker/dist/cjs/checkers/eslint/main.js:122:36)
Emitted 'error' event on Worker instance at:
    at [kOnErrorMessage] (node:internal/worker:290:10)
    at [kOnMessage] (node:internal/worker:301:37)
    at MessagePort.<anonymous> (node:internal/worker:202:57)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:731:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28)

Node.js v18.8.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Reproduction

This is the config I'm using:
```js
export default defineConfig({
    plugins: [
        vue(),
        checker({
            overlay: {
                initialIsOpen: false,
                position: 'br',
            },
            enableBuild: false,
            vueTsc: {
                tsconfigPath: 'tsconfig.vitest.json',
            },
            eslint: {
                lintCommand: 'eslint . --ext .vue,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore',
            },
        }),
    ],
})
```
Current versions that are used:
```
eslint@8.34.0
vite@4.1.4
vite-plugin-checker@0.5.6
```

Not sure if any additional information is required, I would gladly provide that

### Expected behavior

Until recently this worked without issues, since I updated to `vite-plugin-checker@0.5.6` and `vue-tsc@^1.1.0` I noticed the issue, but I'm not sure if that actually causes the issue

Running eslint manually works without issues and shows warnings like these:
```
npx eslint . --ext .vue,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore

/[PATH_TO_PROJECT]/src/somefile.ts
   33:5    warning  Missing accessibility modifier on method definition start  @typescript-eslint/explicit-member-accessibility
   33:128  warning  Unexpected any. Specify a different type                         @typescript-eslint/no-explicit-any
```

### System Info

```shell
System:
    OS: macOS 13.2.1
    CPU: (8) arm64 Apple M1
    Memory: 62.25 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.8.0 - ~/Library/Caches/fnm_multishells/1337_1677143203968/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 8.18.0 - ~/Library/Caches/fnm_multishells/1337_1677143203968/bin/npm
  Browsers:
    Chrome Canary: 112.0.5613.0
    Firefox Developer Edition: 111.0
    Safari: 16.3
  npmPackages:
    vite-plugin-checker: ^0.5.5 => 0.5.6
```
```


### Additional context

_No response_

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

--- a/packages/vite-plugin-checker/src/checkers/eslint/main.ts
+++ b/packages/vite-plugin-checker/src/checkers/eslint/main.ts
@@ -1,6 +1,7 @@
 import chokidar from 'chokidar'
 import { ESLint } from 'eslint'
 import path from 'path'
+import invariant from 'tiny-invariant'
 import { fileURLToPath } from 'url'
 import { parentPort } from 'worker_threads'
 
@@ -38,6 +39,11 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
       if (!pluginConfig.eslint) return
 
       const options = optionator.parse(pluginConfig.eslint.lintCommand)
+      invariant(
+        !options.fix,
+        'Using `--fix` in `config.eslint.lintCommand` is not allowed in vite-plugin-checker, you could using `--fix` with editor.'
+      )
+
       const translatedOptions = translateOptions(options) as ESLint.Options
 
       const logLevel = (() => {

